frontend/src/metabase/collections/components/CollectionSidebar/CollectionSidebar.styled.tsx

```diff
@@ -52,9 +56,39 @@ export const Sidebar = styled.aside`
 
 export const SidebarHeading = styled.h4`
   color: ${color("text-medium")};
+  font-size: 12px;
+  letter-spacing: 0.5px;
   font-weight: 700;
   margin-left: ${space(3)};
   text-transform: uppercase;
+  user-select: none;
+
+  &:hover {
+    color: ${color("text-dark")};
```

Comment:
I added this here but it should really only apply to collection headings that are clickable/collapsable. Wasn't sure how to apply that conditional.

Reply:
Thanks @mazameli. Moved it into the `onClick` prop block.